custom_components/tuya_local/devices/madimack_elite_v3_heatpump.yaml

```diff
@@ -76,6 +76,20 @@ primary_entity:
       name: unknown_107
       type: boolean
 secondary_entities:
+  - entity: binary_sensor

```

Comment:
You should remove the "unknown_15" attribute from above, since it is now identified.

Reply:
Removed the unknown entry!